browse/static/js/feedbackOverlay.js

```diff
@@ -676,4 +676,7 @@ document.addEventListener("DOMContentLoaded", () => {
     });
 
     document.getElementById('myFormContent').onsubmit = submitBugReport;
+
```

Comment:
If this is going to override the user's browser default, I don't think we should do this.

Reply:
Why not? This is convenient for the user because it allows us to prefill the github issue template for them. It also lets us put the data into the feedback table at the same time